doc/dev/jax_integration.rst

```diff
@@ -88,8 +108,10 @@ that doesn't work with Catalyst includes:
 
 - ``jax.numpy.polyfit``
 - ``jax.numpy.fft``
-- ``jax.numpy.ndarray.at[index]`` when ``index`` corresponds to all array
-  indices.
+- ``jax.numpy.ndarray.at[index]`` when ``index`` corresponds to multiple
+  indices. Note that updating multiple indices is supported as long as the
+  indices are sorted and unique, and the appropriate parameters are provided
+  (see the :ref:`jax-sharp-bits` section for details).
```

Comment:
I think the previous entry is important to preserve, unless we support supplying all indices now? (can't remember)

For the new entry:
```suggestion
- ``jax.numpy.ndarray.at[index]``  when ``index`` corresponds to all array
  indices.
- ``jax.numpy.ndarray.at[index]``  when ``index`` contains multiple indices
  *and* the indices overlap, or are not sorted in ascending order (see the
  :ref:`jax-sharp-bits` section for details).
```

Reply:
Neat 👍